cfgov/prepaid_agreements/jinja2/prepaid_agreements/index.html

```diff
@@ -0,0 +1,100 @@
+{% extends 'layout-full.html' %}
+
+{# HEAD items #}
+
+{% block title -%}
+    Prepaid Agreements Search | Consumer Financial Protection Bureau
+{%- endblock title %}
+
+{% block css -%}
+    {{ super() }}
+    <link rel="stylesheet" href="{{ static('apps/prepaid-agreements/css/main.css') }}">
+{%- endblock css %}
+
+{# BODY items #}
+
+{% block content_main_modifiers -%}
+    content__1-3 prepaid-agreements prepaid-agreements-search
+{%- endblock %}
+
+{% block content_main scoped %}
+    {%- import 'molecules/breadcrumbs.html' as breadcrumbs with context -%}
+    {% set breadcrumb_items = [{
+        'href': '/data-research/prepaid-agreements/',
+        'title': 'Prepaid account agreements'
+    }]%}
+    {{ breadcrumbs.render(breadcrumb_items) }}
+
+    <div class="content_wrapper search_form">
+        <div class="block block__sub search_intro">
+            <h1>Prepaid account agreements database</h1>
+            <div class="lead-paragraph u-readable-width">
+                Find prepaid product agreements from more than 80 issuers. You can search agreements by issuer name, product name, prepaid account type, and other critera.
+            </div>
+            <p>Note that we are unable to do a full-text search of agreement files.</p>
+            <p>You may also choose to download all prepaid agreements as a <a href="https://files.consumerfinance.gov/a/assets/prepaid-agreements/bulk.zip">single compressed file</a> (1.3 GB, updated nightly).
+        </div>
+
+        <div class="search_wrapper u-mt30">
```

Comment:
Do you need the margin utilities here and the next link or should `block` classes be used?

Reply:
Updated in https://github.com/cfpb/cfgov-refresh/pull/5207/commits/7371cd90fac1928b166a496101f3bb76ed858b96